packages/system/src/MasterList/ListView/ListView.tsx

```diff
@@ -7,13 +7,15 @@ import {
   useOmSupplyApi,
   useListData,
   useNavigate,
+  useHostContext,
 } from '@openmsupply-client/common';
 import { Toolbar } from './Toolbar';
 import { getMasterListListViewApi } from './api';
 import { AppBarButtons } from './AppBarButtons';
 import { MasterListRow } from '../types';
 
 export const MasterListListView: FC = () => {
+  const { store } = useHostContext();
```

Comment:
```suggestion
  const { store } = useAuthContext();
```

yeah, I plan to update them all anyway, so not a biggie. more concerned that we'll copy paste the wrong version.  I think the distinction between host and auth concerns is important, better to have the auth version used when using for auth purposes

Reply:
Yup, done!